lib/ColorPicker-view.coffee

```diff
@@ -357,9 +357,10 @@
             # the middle of the selection range
             # TODO: There can be lines over more than one row
             if _match
-                _selectionPosition = _selection.marker.getPixelRange()
+                {left, width} = Editor.pixelRectForScreenRange(_selection.getScreenRange())
+                right = left + width
                 _cursorPosition = Cursor.getPixelRect()
-                _cursorPosition.left = _selectionPosition.end.left - ((_selectionPosition.end.left - _selectionPosition.start.left) / 2)
+                _cursorPosition.left = right - ((right - left) / 2)
```

Comment:
Why not just `_cursorPosition.left = right - (width / 2)`? :) Isn't width the same as (right - left)? because right = left + width... Maybe i'm wrong again... :confused: